{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/testrunner.py

```diff
@@ -0,0 +1,14 @@
+from django.test.runner import DiscoverRunner
+
+
+class TestRunner(DiscoverRunner):
+    def __init__(self, *args, **kwargs):
+        tags = kwargs.get("tags")
+        exclude_tags = kwargs.get("exclude_tags")
```

Comment:
maybe have this default to a list so that you can just append to it regardless and don't need the two ifs